staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -207,6 +207,17 @@ message Mount {
     MountPropagation propagation = 5;
 }
 
+// IDMapping describes host to container ID mappings for a pod sandbox.
```

Comment:
We shouldn't update v1alpha2 with new features anymore.

Reply:
The CI was failing with https://github.com/kubernetes/kubernetes/commit/89406bc475dd348b97226f6fbbdf1847b242751d

Reply:
should I revert to the old version?